.gitignore

```diff
@@ -10,3 +10,5 @@ website/build/
 website/yarn.lock
 website/node_modules
 website/i18n/*
+
+.idea/**
```

Comment:
Let's not commit this - the `.idea` folder can be ignored globally in your local git settings

Reply:
Removed